### PR TITLE
chore(op-plasma-eigenda): Remove dead and unecessary code

### PR DESCRIPTION
**Pull request by epociask**
Closes: https://github.com/Layr-Labs/op-plasma-eigenda/issues/4 , #7

### CODE REVIEW

File: .gitmodules
Comment (teddyknox) [inline .gitmodules:1]: Nice, I like this approach.

File: README.md [hunk 1]
```diff
@@ -49,17 +38,29 @@ type Cert struct {
 }
 ```
 
+**NOTE:** Commitments are cryptographically verified against the data fetched from EigenDA for all `/get` calls.
```
Comment (teddyknox) [inline README.md:41]: 👍

File: commitment.go [hunk 2]
```diff
@@ -19,6 +19,16 @@ var ErrCommitmentMismatch = errors.New("commitment mismatch")
 // CommitmentType is the commitment type prefix.
 type CommitmentType byte
 
+// Max input size ensures the canonical chain cannot include input batches too large to
+// challenge in the Data Availability Challenge contract. Value in number of bytes.
+// This value can only be changed in a hard fork.
+const MaxInputSize = 130672
```
Comment (teddyknox) [inline commitment.go:25]: This is just a sanity check right? I would think OP Stack should be enforcing this invariant.

**Review by teddyknox — APPROVED** [review]
LGTM

File: commitment.go (hunk 2 above)
Comment (epociask) [reply commitment.go:25]: You are correct - left over copy pasta

Comment (epociask) [reply commitment.go]: [a28ca7e](https://github.com/Layr-Labs/op-plasma-eigenda/pull/8/commits/a28ca7efd1cb66276ac9d0d20cf605a1f43a9b01)

**Pull request merged**